restcomm/restcomm.commons/src/main/java/org/restcomm/connect/commons/faulttolerance/RestcommSupervisor.java

```diff
@@ -0,0 +1,111 @@
+package org.restcomm.connect.commons.faulttolerance;
+
+import akka.actor.ActorContext;
+import akka.actor.ActorPath;
+import akka.actor.ActorRef;
+import akka.actor.ChildRestartStats;
+import akka.actor.OneForOneStrategy;
+import akka.actor.Props;
+import akka.actor.StopChild;
+import akka.actor.SupervisorStrategy;
+import akka.actor.Terminated;
+import akka.actor.UntypedActor;
+import akka.event.Logging;
+import akka.event.LoggingAdapter;
+import akka.japi.Function;
+import scala.collection.Iterable;
+import scala.concurrent.duration.Duration;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+import static akka.actor.SupervisorStrategy.resume;
+
+/**
+ * Created by gvagenas on 22/02/2017.
+ */
+public class RestcommSupervisor extends UntypedActor {
```

Comment:
This class will have the default fault-tolerance strategy which is to restart on unhandled exceptions. 
Makes sense to have the postStop to print some info